### ISSUE TITLE
Using the library under a path (homepage in package.json)

### ISSUE THREAD

**Issue by tomaluca95** [AxaGuilDEv/react-oidc#566]
### Issue and Steps to Reproduce

Create a new react project and set `homepage` value in the `package.json`

The libary redirects to `/authorization/` without the prefix.

This is useful to deploy an application avoiding CORS, for example inside `/ui/`

### Versions

3.1.6

#### Expected

Redirect inside the context defined in the `package.json`

#### Actual

Redirect in the root

**Comment by youf-olivier** [comment]
Thanks for the issue, sorry we were a bit overwhelmed. I'll look all the issues soon

**Comment by briandonahue** [comment]
I created #586 before noticing this issue (I was searching for the wrong terms before posting apparently!).  There's a little more context in mine.  @tomaluca95 did you have any success with workarounds for the issue?

I used absolute URLs in my OIDC Configuration to workaround the issue you mention
```
    redirect_uri: `${window.location.origin}${baseUrl}/authentication/callback`,
    silent_redirect_uri: `${window.location.origin}${baseUrl}/authentication/silent_callback`,
 ```

but then there is still an issue when `react-oidc` tries to redirect the authenticated user back to the original target URL.
